ophyd/ophydobj.py

```diff
@@ -61,8 +72,15 @@ def root(self):
 
     @property
     def report(self):
+        '''A report on the object.'''
         return {}
 
+    @property
+    def event_types(self):
+        '''Events that can be subscribed to via `obj.subscribe`
+        '''
+        return self._pub_event_types
```

Comment:
I could be talked out of exposing this but on several occasions have really wanted this, at least as a debugging / dev tool.

Will turn them into tuples at definition (good catch).

Is 'event_types' likely to squat on something from EPICS?